js/package.json

```diff
@@ -22,10 +22,10 @@
     "@mantine/hooks": "7.10.2",
```

Comment:
nit: Commit message is what gets merged into main with rebase, so the JsInfra should go in there as well.